director/accounts/migrations/0022_auto_20200527_0112.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.12 on 2020-05-27 01:12
```

Comment:
It seems like we should be able to combine these into a single migration:

director/accounts/migrations/0019_account_user.py
director/accounts/migrations/0020_auto_20200527_0059.py
director/accounts/migrations/0021_auto_20200527_0111.py
director/accounts/migrations/0022_auto_20200527_0112.py

Reply:
Yes, I think you're right there. Will do that.

Reply:
Have resolved in e42c859